Doc generation fails on Windows

This seems to be only on Windows. I have v10. The root of the issue comes from compodoc but thought I would start here to see if there are changes we can make to the seed.


```..\angular-seed\angular-seed\node_modules\.bin\compodoc:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
    at createScript (vm.js:74:10)
    at Object.runInThisContext (vm.js:116:10)
    at Module._compile (module.js:533:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "<projpath>\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "compodoc"
```

--- a/package.json
+++ b/package.json
@@ -40,8 +40,8 @@
     "tests.all": "npm test && npm run e2e.ci",
     "webdriver-start": "node ./node_modules/protractor/bin/webdriver-manager start",
     "webdriver-update": "node ./node_modules/protractor/bin/webdriver-manager update",
-    "compodoc": "node ./node_modules/.bin/compodoc -p src/client/tsconfig.json",
-    "serve.compodoc": "node ./node_modules/.bin/compodoc -s"
+    "compodoc": "./node_modules/.bin/compodoc -p src/client/tsconfig.json",
+    "serve.compodoc": "./node_modules/.bin/compodoc -s"
   },
   "author": "Minko Gechev <mgechev>",
   "license": "MIT",

